src/pages/Error/Error.test.tsx

```diff
@@ -0,0 +1,11 @@
+import Error from './Error';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
```

Comment:
this import is not needed

Reply:
looked it over when I cleaned it, thanks. deleted